public/index.html

```diff
@@ -24,7 +24,13 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <!-- Bootstrap JS Popper bundle -->
```

Comment:
JS Bundle needed for tooltips and modals

Reply:
Does this script need to be incuded inside the head?

Reply:
thx man!! this happens when you do 3 things at the same time 😄